hybrid/dispatch/power_storage/linear_voltage_convex_battery_dispatch.py

```diff
@@ -19,14 +19,14 @@ def __init__(self,
                  system_model: BatteryModel.BatteryStateful,
                  financial_model: Singleowner.Singleowner,
                  block_set_name: str = 'convex_LV_battery',
-                 include_lifecycle_count: bool = True,
+                 dispatch_options: dict = {},
```

Comment:
Setting mutable defaults for arguments is dangerous in Python, I'd recommend defaulting to None and initializing inside the method. For example:

```python
def __init__(self, ..., dispatch_options=None):
    if dispatch_options is None:
        dispatch_options = {}
    ...
```

Reply:
Thanks fixed